src/contentmanager.cpp

```diff
@@ -619,8 +619,12 @@ QStringList ContentManager::getDownloadIds()
     return list;
 }
 
-void ContentManager::setCurrentLanguage(QStringList languageList)
+void ContentManager::setCurrentLanguage(QList<QPair<QString, QString>> langPairList)
```

Comment:
This `QList<QPair<QString, QString>>` is a bit ugly (and everywhere).
We should rename it to something more meaningful. I suggest would `FilterList` (until I realize we have already `SelectionList` which does the same 😄 )

But we can keep a different name to help us remember that it may be different kind of things:
- FilterList : A list of (potential) filters.
- SelectionList: A list of selected filters.

Reply:
Done :>